### DISCUSSION TITLE
Postal Email Pending

### DISCUSSION THREAD

**Discussion by uross98** [postalserver/postal#1889] [Help with using Postal]
Hello im trying out Postal and im having problems right now 

So far after `postal stop` and `postal star` 
`postal status` State of everything is Up
But after few minutes i get

Postal Requierer_1 and Worker_1 Code Exit 1

All DNS records are fine and im not a pro but postal.yml should be fine 

Help would be greatly appreciated

**Comment by reetikpant** [comment]
dm me on telegram @justqax

**Reply by robertpreshyl** [reply]
i sent you a dm on Tg

**Comment by willpower232** [comment]
You'll need to use `postal logs requeuer` to see what the problem is. At a rough guess, I'd imagine that the RabbitMQ is either misconfigured or takes longer to start than Postal does. What happens if you `postal start` again without doing `postal stop`?
